packages/angular/README.md

```diff
@@ -31,9 +31,7 @@ Include either light or dark theme in your application, for example in a newly i
       "architect": {
         "build": {
           "options": {
-            "styles": [
-              "@synergy-design-system/tokens/themes/light.css"
-            ]
+            "styles": ["@synergy-design-system/tokens/themes/light.css"]
```

Comment:
I would leave it in the more verbose way in the docs, as it is easier for copy&paste when the setting is already there.

Reply:
Done.